minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterImpl.java

```diff
@@ -581,13 +581,16 @@ public synchronized void start() throws IOException, InterruptedException {
         }
 
         log.warn("Initializing ZooKeeper");
-        try (var vm = VolumeManagerImpl.get(siteConfig, config.getHadoopConfiguration())) {
+        var hadoopConfig = new Configuration(false);
+        File csFile = new File(config.getConfDir(), "core-site.xml");
+        hadoopConfig.addResource(csFile.toURI().toURL());
+
+        try (var vm = VolumeManagerImpl.get(siteConfig, hadoopConfig)) {
           log.info("Calling init with {}", args);
           Initialize init = new Initialize();
           Initialize.Opts opts = new Initialize.Opts();
           opts.parseArgs("accumulo init", args.toArray(new String[0]));
-          InitialConfiguration initConfig =
-              new InitialConfiguration(config.getHadoopConfiguration(), siteConfig);
+          InitialConfiguration initConfig = new InitialConfiguration(hadoopConfig, siteConfig);
```

Comment:
Any benefit to logging the config (at debug level?) here to help show what it is running with (or not running if it fails?)

Reply:
I think I figured it out. The ones that override `configureMiniCluster()` will write out a core-site.xml. The other ones don't so I just need to check if the file exists.

Reply:
To be clear - I was asking if it would help in the future - so that if there are other problems, it is always available to help see what the configuration is, rather than guessing.  It could be being logged elsewhere, I do not know.  Figured you'd be running the tests and you would be in a position to evaluate if something would help, if it is not available now.